sql/healthpro/notices.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE `notices` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `url` varchar(255) NOT NULL,
+  `message` text NOT NULL,
+  `full_page` tinyint(1) NOT NULL DEFAULT 0,
+  `start_ts` timestamp NULL DEFAULT NULL,
+  `end_ts` timestamp NULL DEFAULT NULL,
```

Comment:
If we need to add a message of indeterminate expiration, I'm guessing the instruction will be to pick a date far enough in the future?

Side question: Do we ever need to retain these, or would we regularly clean out old messages?

Reply:
Start and end times are optional.  If you put no start or end, it will always be active as long as it is enabled.  If only a start time is entered, it will start at that time and be active indefinitely from that time on.  If only an end time is entered, it will be active immediately until the end time passes.

I imagine that we would delete notices when we are done with them.